k8s/namespaces/wa/wa-workflow-api/wa-workflow-api.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: wa-workflow-api
+spec:
+  releaseName: wa-workflow-api
+  chart:
+    git: git@github.com:hmcts/hmcts-charts
+    ref: master
+    path: stable/wa-workflow-api
```

Comment:
Cannot see this in here : https://github.com/hmcts/hmcts-charts/tree/master/stable